code/__DEFINES/inventory.dm

```diff
@@ -12,4 +12,16 @@
 /// Usually represents objects that require two hands to operate, (e.g. shotgun, two-handed melee weapons)
 #define WEIGHT_CLASS_HUGE 5
 /// Essentially means it cannot be picked up or placed in an inventory, (e.g. mech parts, safe)
-#define WEIGHT_CLASS_GIGANTIC 6
+#define WEIGHT_CLASS_GARGANTUAN 6
```

Comment:
Do not change this define name

Reply:
![immagine](https://github.com/user-attachments/assets/a29e26b7-7552-48d8-95d1-82ca76fe086c)
We aren't

Reply:
OK, I changed it back